packages/venia-ui/lib/queries/addConfigurableProductsToCart.graphql

```diff
@@ -21,8 +21,33 @@ mutation addConfigurableProductToCart(
                 product {
                     name
                     sku
+                    small_image {
```

Comment:
Since we can't use fragments, each of the cart mutations/queries uses this identical response object. That's lame!